protostar.toml

```diff
@@ -0,0 +1,31 @@
+["protostar.config"]
+protostar_version = "0.3.2"
+
+["protostar.project"]
+libs_path = "contracts/lib"
+
+["protostar.contracts"]
```

Comment:
Need to add Admin as a contract? (and rerun `protostar build`)

Reply:
https://github.com/42labs/Empiric/pull/156/commits/f0902f5c22bc90b95cca0db26b14d545fc136f58